cmd/mt-gateway/main.go

```diff
@@ -47,14 +51,27 @@ func main() {
 		fmt.Println("Flags:")
 		flag.PrintDefaults()
 	}
-	flag.Parse()
+
+	// Only try and parse the conf file if it exists
+	path := ""
+	if _, err := os.Stat(*confFile); err == nil {
+		path = *confFile
+	}
+	conf, err := globalconf.NewWithOptions(&globalconf.Options{
+		Filename:  path,
+		EnvPrefix: "MT_",
+	})
+	if err != nil {
+		log.Fatalf("error with configuration file: %s", err)
+		os.Exit(1)
```

Comment:
is the `os.Exit()` reached after `log.Fatalf()`? I thought `log.Fatalf()` should already exit the program

Reply:
You are correct: 2a9acc35ef9331d5cfe286152a6e83b2dad9a108